src/bin/vip.js

```diff
@@ -115,6 +115,9 @@ const rootCmd = async function() {
 			throw e;
 		}
 
+		//de-anonymize user for tracking
+		await aliasUser( token.id );
```

Comment:
We'll need to update the `uuid` in `lib/token` as well, otherwise we'll keep sending the anon ID with future events.

Reply:
Okay, I think the following commit addressed that.